src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -96,41 +136,95 @@ def export_config(cmd,
                   dest_connection_string=None,
                   dest_label=None,
                   # to-app-service parameters
-                  appservice_account=None):
+                  appservice_account=None,
+                  skip_features=False):
     # fetch key values from user's configstore
     src_kvs = __read_kv_from_config_store(
```

Comment:
>__read_kv_from_config_store [](start = 14, length = 27)

same with comment above, let __read_kv_from_config_store to fetch all items and return key-values/features separately

Reply:
As discussed offline, this approach would not work if user specifies key filter like abc*. We need to make two separate calls to get all the keys and features.

---
In reply to: [355747275](https://github.com/Azure/azure-cli/pull/11467#discussion_r355747275) [](ancestors = 355747275)